sig-creation-procedure.md

```diff
@@ -0,0 +1,97 @@
+### SIG creation procedure
+
+* Propose the new SIG publicly, including a brief mission statement, by emailing kubernetes-dev@googlegroups.com and kubernetes-users@googlegroups.com, then wait a couple of days for feedback
+* Create a group (see the detailed instructions below)
+* Ask a repo maintainer to create a github label, if one doesn't already exist: sig/foo
+* If you wish, request a new [kubernetes.slack.com](http://kubernetes.slack.com) channel (#sig-foo) from @sarahnovotny.  New users can join at [slack.kubernetes.io](http://slack.kubernetes.io).
+* Organize video meetings as needed. No need to wait for the [Weekly Community Video Conference](community/README.md) to discuss. Please report summary of SIG activities there.
+ * Request a Zoom account from @sarahnovotny if you expect more than 30 attendees or attendees from China.
+ * Add the meeting to the community meeting calendar by inviting cgnt364vd8s86hr2phapfjc6uk@group.calendar.google.com.
+* Use existing proposal and PR process
+* Announce new SIG on kubernetes-dev@googlegroups.com and ask a repo maintainer to create a kubernetes/community directory and github team for the new group
+* Submit a PR to add any SIG-related docs, schedules, roadmaps, etc. to your new kubernetes/community/SIG-foo directory.
+* Slack activity is archived at [kubernetes.slackarchive.io](http://kubernetes.slackarchive.io).  To start archiving a new channel invite the slackarchive bot to the channel via `/invite @slackarchive`
+
+#### 
+**Google Groups creation**
+
+Create Google Groups at [https://groups.google.com/forum/#!creategroup](https://groups.google.com/forum/#!creategroup), following the procedure: 
+
+* Create a group using the name convention below;
+* Group has to be created as an e-mail list with at least three owners (including sarahnovotny at google.com and ihor.dvoretskyi at gmail.com);
+* To add the owners, visit the Group Settings (drop-down menu on the right side), select Direct Add Members on the left side and add Sarah and Ihor via email address (with a suitable welcome message); in Members/All Members select Ihor and Sarah and assign to an "owner role";
+* Set "View topics", "Post", "Join the Group" permissions to be "Public"
+
+
+Name convention:
+
+* kubernetes-sig-foo-misc
```

Comment:
Please add `* kubernetes-sig-foo` to the beginning of the list. This is the discussion group. The other groups are just for mirroring github notifications. It would be useful to include one sentence about that here.